setup.py

```diff
@@ -0,0 +1,11 @@
+from distutils.core import setup, Extension
+import os
+
+if os.name == 'posix':
+    cflags = ['-std=c99']
+else:
+    cflags = None
+
+pyshoco = Extension('pyshoco', sources = ['pyshoco.c', 'shoco.c'], include_dirs = ['.'], extra_compile_args = cflags)
```

Comment:
The typical Python style is not to put spaces around `=` used for named arguments. You can also put the arguments on separate lines in both of these calls.